lib/views/signin_page/signin_page.dart

```diff
@@ -68,7 +179,20 @@ class SignInPage extends StatelessWidget {
                   ),
                   const Gap(32),
                   TouchableOpacity(
-                    onTap: () {},
+                    onTap: () => context.go(
+                      Routes.alertPage,
+                      extra: {
+                        'type': AlertParams(
+                            type: AlertType.almost,
+                            title: 'Almost there!',
+                            description:
```

Comment:
Don't access the VM directly for functions. use `ref.read(vm). scanQrCode()`;

Reply:
vm here isn't the ViewModel but a reference to the homeVM defined as `ref.read(homeVM);`